.rubocop.yml

```diff
@@ -2,7 +2,7 @@
 inherit_from:
   - ./config/default.yml
   - ./config/rails.yml
-  - ./config/rake.yml
+  - ./config/rspec.yml
```

Comment:
Are you meaning to remove the rake config from here?

Reply:
Ah, forgot to mention it's included [by default](https://github.com/alphagov/rubocop-govuk/blob/master/config/default.yml#L26).